src/commands/lambda/function.ts

```diff
@@ -251,6 +259,23 @@ export const calculateUpdateRequest = (
     changedEnvVars[FLUSH_TO_LOG_ENV_VAR] = settings.flushMetricsToLogs.toString()
   }
 
+  if (settings.environment && oldEnvVars[ENVIRONMENT_ENV_VAR] !== settings.environment.toString()) {
```

Comment:
maybe this chunk of code could be refactored? it seems that the logic for each env variable is the same?

Reply:
Yup, was thinking the same, maybe just a map for every env variable?